packages/components/src/custom-select-control/test/index.js

```diff
@@ -82,10 +82,62 @@ const ControlledCustomSelectControl = ( {
 };
 
 describe.each( [
-	[ 'uncontrolled', UncontrolledCustomSelectControl ],
-	[ 'controlled', ControlledCustomSelectControl ],
+	[ 'Uncontrolled', UncontrolledCustomSelectControl ],
+	[ 'Controlled', ControlledCustomSelectControl ],
 ] )( 'CustomSelectControl %s', ( ...modeAndComponent ) => {
-	const [ , Component ] = modeAndComponent;
+	const [ mode, Component ] = modeAndComponent;
+
+	it( 'Should select the first option when no explicit initial value is passed without firing onChange', () => {
+		const mockOnChange = jest.fn();
+		render( <Component { ...props } onChange={ mockOnChange } /> );
+
+		expect(
+			screen.getByRole( 'button', {
+				expanded: false,
+			} )
+		).toHaveTextContent( 'violets' );
```

Comment:
Same as above in terms of whether to use a string literal or reference the test fixtures like we do in some of the tests:

```suggestion
		).toHaveTextContent( props.options[ 0 ].name );
```